rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+### **DCP PR:**
+
+*Leave this blank until the RFC is approved then the Author(s) must
+create a link between the assigned RFC number and this pull request in
+the format:*
+
+[dcp-community/rfc\#](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)
+
+**Informational RFC: Managed Access Data in the Datas Control Platform**
+========================================================================
+
+**Summary**
+-----------
+
+This RFC describes the requirements for the DCP to support storing and
+controlling access to managed access data.
+ 
+**Author(s)**
+-------------
+
+*Recommended format for Authors:*
+
+[Kevin
+Osborn]([[mailto:kosborn2\@ucsc.edu]](mailto:kosborn2@ucsc.edu))
+
+[Trent
+Smith]([[mailto:trent.smith\@chanzuckerberg.com]](mailto:trent.smith@chanzuckerberg.com))
+
+[Brian
+Hannafious]([[mailto:bhannafi\@ucsc.edu]](mailto:bhannafi@ucsc.edu))
+
+[Brian
+O'Connor]([[mailto:broconno\@ucsc.edu]](mailto:broconno@ucsc.edu))
+
+**Shepherd**
+------------
+
+[Name](mailto:username\@example.com)
```

Comment:
Add a Shepherd (which could be one of the authors).

Reply:
I put Brian here. I will ask for forgiveness when he gets back on Thursday.